src/utils/paymentQRHelper.test.ts

```diff
@@ -49,70 +49,123 @@ describe("tests for findValueByKey", () => {
 });
 
 describe("tests for getUpdatedTransactionsPaymentQRIdentifiers", () => {
```

Comment:
Can be done later in another PR, but what do you feel about splitting this describe block into 2 describe blocks? One will be for non nets, and one will be for nets. 

I feel it might help with readability haha

Reply:
Btw, the tests are very clear, and I can understand them easily. Nice job ahah

Reply:
Great suggestion, have split the tests into separate `describe` blocks in [e876718](https://github.com/rationally-app/mobile-application/pull/532/commits/e87671860f7ffbb1118f3e28032c6c00ee85a4bf)!